crates/flipperzero/src/dialogs/mod.rs

```diff
@@ -5,10 +5,10 @@ use alloc::ffi::CString;
 
 use core::ffi::{c_char, c_void, CStr};
 use core::marker::PhantomData;
+use core::mem::MaybeUninit;
 use core::ptr::{self, NonNull};
-
```

Comment:
Undo the deletion of this newline (it is intentionally separating the Rust core changes from the external dependencies).

Reply:
Reverted